flink-java/src/test/java/org/apache/flink/api/java/sca/UdfAnalyzerTest.java

```diff
@@ -109,7 +112,7 @@ public String map(MyPojo value) throws Exception {
 	@Test
 	public void testForwardWithGenericTypePublicAttrAccess() {
 		compareAnalyzerResultWithAnnotationsSingleInput(MapFunction.class, Map4.class,
-				"org.apache.flink.api.java.sca.UdfAnalyzerTest$MyPojo", "String");
+			TypeInformation.of(new TypeHint<GenericTypeInfo<MyPojo>>(){}), Types.STRING);
```

Comment:
The `GenericTypeInfo` here seems to be not correct.

Reply:
Then the code needs to be `new GenericTypeInfo<>(MyPojo.class)`. Otherwise it will try and determine why the Type Info for `GenericType` is and it just happens to be a generic type ;-)